snap/quota/resources.go

```diff
@@ -132,6 +151,26 @@ func (qr *Resources) validateThreadQuota() error {
 	return nil
 }
 
+func (qr *Resources) validateJournalQuota() error {
+	// Journal quota is a bit different than the rest, we allow nil values
+	// for the size and rate, because this means that the only 'quota' we want
+	// in this case is to scope the journal for snaps in the group to just a namespace
+	// without any extra limits.
+	// XXX: Should we have a minimum log size that can be set?
```

Comment:
Yes, I think it makes sense, absurdly low values should probably be rejected; does systemd doc says anything about that?

Reply:
I can only find a maximum usage of 4GB, and it seems in the systemd source code that the minimum usage can't be lower than current usage to avoid it vacuuming files.

Reply:
My gut feeling is that we can start with something like 64k as the lower limit and then just go lower if we have use-cases. It will also help to protect people from doing silly things like setting it to `32` instead of `32kb`.

Reply:
Agreed, i'll set it to 64kb!

Reply:
@mvo good point about silly mistakes like 32 vs 32kb! 64kb sounds good